guardian/testapp/testsettings.py

```diff
@@ -51,6 +51,13 @@
 
 DATABASES = {'default': env.db(default="sqlite:///")}
 
+# workaround - "Illegal mix of collations (utf8_unicode_ci,IMPLICIT) and (utf8_general_ci,IMPLICIT) for operation '='"
+if 'mysql' in DATABASES['default']['ENGINE']:
+    DATABASES['default']['TEST'] = {
+        'CHARSET': "utf8",
+        'COLLATION': "utf8_general_ci",
+    }
```

Comment:
It would be safer to extend the introduced `django.VERSION >= (1, 11)` checks as follows:
```python
from django.conf import settings
if django.VERSION >= (1, 11) and all('mysql' not in db['ENGINE'] for db in settings.DATABASES.values()):
    pass
```